modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -466,6 +466,8 @@ x-is-a-reserved-word={0} is a reserved word and cannot be used.
 x-key={0} Key
 x-removed={0} (Removed)
 x-version-x={0} ({1})
+xls-warning-cell-chars-limit=The total number of characters that a cell can contain is 32,767 characters. In case of this limit is exceeded cell content will be trimmed.
+xls-warning-message=This XLS file contains user supplied inputs. Opening a XLS file in a spreadsheet program may be dangerous.
```

Comment:
@samanzanedo, for this specific pull, can we keep only the first warning message regarding the cell maximum length?

Besides, can you please split this commit into two commits? The first one should add only the new language key and the second commit should have only the execution of the buildLang. This is pattern expected by Brian. :slightly_smiling_face: 

Lastly, can you please rename the language key according [this example](https://github.com/liferay-forms/liferay-portal/blob/978cdc632e8824e9480b8e47a3e400f92612fd06/modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties#L61)? This is the common pattern used although some old language keys aren't following it.